### PR TITLE
feat: added color adjustment feature

### LINKED ISSUE #380: Adjust color balance
### Is your feature request related to a problem?

Adjusting the color balance of an image is another common operation.

### Desired solution

* New method `adjust_color_balance` on `Image`
* `factor` parameter to adjust the strength of the change
* Use [`ImageEnhance.Color`](https://pillow.readthedocs.io/en/stable/reference/ImageEnhance.html#PIL.ImageEnhance.Color) internally
* Warn if `factor` is `1` since this leaves the image unchanged
* Raise error if `factor` is below 0

### Possible alternatives (optional)

_No response_

### Screenshots (optional)

_No response_

### Additional Context (optional)

_No response_

### PR DESCRIPTION
**Pull request by patrikguempel**
Closes #380

### Summary of Changes
Added the color adjustment feature analogous to the previous image tasks.

### CODE REVIEW

**Review by daniaHu — DISMISSED** [review]
[[LGTM](https://lgtm.party/p/P6FK0QO/)](url)

**Review by jxnior01 — APPROVED** [review]
Nice work!

File: src/safeds/data/image/containers/_image.py [hunk 1]
```diff
@@ -334,6 +334,20 @@ def adjust_contrast(self, factor: float) -> Image:
         image_copy._image = ImageEnhance.Contrast(image_copy._image).enhance(factor)
         return image_copy
 
+    def adjust_color_balance(self, factor: float) -> Image:
```
Comment (Marsmaennchen221) [inline src/safeds/data/image/containers/_image.py:337]: Please add docstrings

**Review by Marsmaennchen221 — CHANGES_REQUESTED** [review]

File: src/safeds/data/image/containers/_image.py [hunk 2]
```diff
@@ -335,6 +335,21 @@ def adjust_contrast(self, factor: float) -> Image:
...
+        ----------
+        factor: float
+            If factor > 1, increase color balance of image.
+            If factor = 1, no changes will be made.
+            If factor < 1, make image greyer.
+            Has to be bigger than or equal to 0.
+
+        Returns
+        -------
+        The new, adjusted image.
```
Comment (Marsmaennchen221) [inline src/safeds/data/image/containers/_image.py:351]:
```suggestion
        image: Image
            The new, adjusted image.
```

**Review by Marsmaennchen221 — CHANGES_REQUESTED** [review]

**Review by Marsmaennchen221 — APPROVED** [review]

**Pull request merged**